src/ClientData/include/ClientData/ProcessData.h

```diff
@@ -79,6 +79,8 @@ class ProcessData final {
     return FindModuleByPath(module_path).has_value();
   }
 
+  [[nodiscard]] bool IsModuleLoaded(const ModuleData* module) const;
```

Comment:
Since this is a member function of ProcessData, it checks whether the module is loaded by this process. 
My suggestion to make it more readable: `IsModuleLoadedByProcess()`, wdyt?

Reply:
Okay, thanks for the explanation. I think your suggestion is better, but I could also live with the current name. However, I would definitely like to see the comment at the usage below (with the assumption stated).

Reply:
I changed the name to `IsModuleLoadedByProcess()` and added a comment in OrbitApp